src/networking/address.rs

```diff
@@ -1,25 +1,20 @@
-/// An enum containing both ipv6 and ipv4 addresses
-
-#[derive(Debug)]
-pub enum IPVersion {
-  IPV4,
-  IPV6,
```

Comment:
This is due to ipv8 not supporting ipv6 on some level like you mentioned right?

If so any reason this implementation should not support it?

Reply:
i mostly removed them because the builtin ip address struct is great in rust. but i removed ipv6 support for now until we get to talk to johan again. Their implementation of it is a bit weird and we will need to talk about it